srcpkgs/muse/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'muse'
+pkgname=muse
+version=3.1.1
+revision=1
+build_style=cmake
+build_helper=qmake
+configure_args="-DENABLE_RTAUDIO=OFF -DLIB_INSTALL_DIR=/usr/lib
+ $(vopt_bool alsa ENABLE_ALSA) $(vopt_bool python ENABLE_PYTHON)"
+hostmakedepends="pkg-config qt5-host-tools qt5-qmake"
+makedepends="$(vopt_if alsa alsa-lib-devel) dssi-devel fluidsynth-devel
+ jack-devel ladish-devel ladspa-sdk libinstpatch-devel liblo-devel liblrdf-devel libsamplerate-devel
+ libsndfile-devel lilv-devel lv2 rubberband-devel qt5-svg-devel qt5-tools-devel
+ $(vopt_if python python3-devel)"
+depends="desktop-file-utils qt5-svg $(vopt_if python 'python3-six python3-Pyro4')"
+short_desc="MIDI/Audio sequencer with recording and editing capabilities"
+maintainer="Evgeny Ermakov <evgeny.v.ermakov@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://muse-sequencer.github.io"
+_name="${pkgname}-${version}"
+distfiles="https://github.com/muse-sequencer/${pkgname}/releases/download/${_name//[-.]/_}/${_name}.tar.gz"
```

Comment:
1. please expand `pkgname`
2. why don't you fetch `https://github.com/muse-sequencer/muse/archive/muse_3_1_1.tar.gz` ? (the tagged commit instead of their uploaded file)

Reply:
> why don't you fetch https://github.com/muse-sequencer/muse/archive/muse_3_1_1.tar.gz

This archive has a different directory structure (also notice the directory name)\:
```sh
$ ll masterdir/builddir/muse-muse_3_1_1/
total 260K
-rw-r--r--  1 evg evg 1.1K Jul 18  2020 README.md
drwxr-xr-x 23 evg evg 4.0K Jul 18  2020 muse3
-rw-r--r--  1 evg evg 249K Jul 18  2020 muse_arranger_mixer.png
```
So I had to set `wrksrc`:
```sh
wrksrc="muse-muse_${version//[-.]/_}"
```
But in any case, an error like this occurs:
```
=> muse-3.1.1_1: running do_configure ...
CMake Error: The source directory "/builddir/muse-muse_3_1_1" does not appear to contain CMakeLists.txt.
```
To me, this will complicate things. So what is your opinion?

Reply:
> please expand `pkgname`

Done.

Reply:
you need `build_wrksrc=muse3` as well

Reply:
Ah, thanks, it works now.

Reply:
But not for all architectures.

Reply:
(solved)